src/pages/developers/index.tsx

```diff
@@ -176,6 +176,10 @@ const StyledCard = styled(Card)`
   box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
   margin: 1rem;
   padding: 1.5rem;
+  justify-content: space-between;
+  h3 {
+    min-height: 67px !important;
+  }
```

Comment:
Why do we need this 🤔?

Reply:
Its because, we are setting the minimum height for the h3 tag(card heading). 
as we are having one line and two line headings both are taking different heights. to equate these, we have set a minimum height as the height of a two line heading.

Reply:
I don't understand what is the issue here and how this would fix it. Could you share a link or an image where we can see the problem?